Classes/Controller/AdministrationController.php

```diff
@@ -166,7 +174,11 @@ protected function invokeModuleController()
         $request->setControllerExtensionName(ucfirst($activeModuleDescription['extensionKey']));
         $request->setControllerName($activeModuleDescription['controller'] . 'Module');
         $request->setControllerActionName('index');
-        $request->setArgument('site', $this->site);
+
+        if( !is_null($this->site) ) {
```

Comment:
Is this PSR-2? You can setup PhpStorm to check/correct/produce PSR-2 compliant code automatically.